strapi/strapi
# fix(content-manager): populate media and nested components on cloning

### What does it do?

Fix the issue we had when we tried to clone an entry with assets, dynamic zones and components, because were not cloned

### Why is it needed?

We want to populate the clone call with the media attributes and components

### How to test it?

- Create an entry with a media attribute or nested components
- try to clone this entry and check all the fields are cloned

### Related issue(s)/PR(s)

Resolves [18403](https://github.com/strapi/strapi/issues/18403)
